### DISCUSSION TITLE
Is the version deployed at table.sadmn.com really the latest?

### DISCUSSION THREAD

**Discussion by tecoad** [sadmann7/tablecn#623] [Q&A] [answered]
Hi @sadmann7,

Just following up with this project, this commit https://github.com/tecoad/shadcn-table/commit/beed202f0942868e77cba5e9aff61cdcf4a445eb introduced a change on the hook, specifically:

```
		getFilteredRowModel: enableAdvancedFilter ? undefined : getFilteredRowModel(),
		getFacetedRowModel: enableAdvancedFilter ? undefined : getFacetedRowModel(),
		getFacetedUniqueValues: enableAdvancedFilter ? undefined : getFacetedUniqueValues(),
```

The thing is, in my fork, the advanced faceted filters are not showing the counts:

![Screenshot 2024-10-16 at 13 43 46](https://github.com/user-attachments/assets/03c39b9a-fd44-4c6b-85c2-7faa2abdba8f)

```
{option.withCount && column?.getFacetedUniqueValues()?.get(option.value) && (
	<span className="ml-auto flex size-4 items-center justify-center font-mono text-xs">
		{column?.getFacetedUniqueValues().get(option.value)}
	</span>
)}
```

Since getFacetedUniqueValues on enableAdvancedFilter is undefined, how is this mean to show the values counter for the advanced toolbar?
The version here https://table.sadmn.com/ is showing it though.

![Screenshot 2024-10-16 at 13 42 45](https://github.com/user-attachments/assets/5d342bb6-c157-429b-bbb7-518ae8a7714b)

Can you confirm? Any reason for this?

**Comment by sadmann7** [comment, marked as answer]
need to update the advanced filter

the advanced filter was quite buggy, will update it with all the fixes

**Reply by tecoad** [reply]
yes, i just noticed it @sadmann7 
For example, if you remove the context and hardcode  enableAdvancedFilter: true, the counter on faceted filter won't work.
Its quite surprisingly that it is working actually, because of the context initial null default state.

**Comment by sadmann7** [comment]
updated the advanced filter.

right now faceted filter count showd the actual item count on the db
